lib/astunparse/unparser.py

```diff
@@ -29,7 +29,7 @@ class Unparser:
     output source code for the abstract syntax; original formatting
     is disregarded. """
 
-    def __init__(self, tree, file=sys.stdout):
+    def __init__(self, tree, file = sys.stdout):
```

Comment:
```suggestion
    def __init__(self, tree, file=sys.stdout):
```

Reply:
Just keeping in sync with original cpython file there